twisted/conch/ssh/keys.py

```diff
@@ -39,6 +42,47 @@
 from twisted.python.deprecate import deprecated, getDeprecationWarningString
 from twisted.python.versions import Version
 
+# Curve lookup table
+curveTable = {
+        'nistp256' : ec.SECP256R1(),
+        'nistp384' : ec.SECP384R1(),
+        'nistp521' : ec.SECP521R1(),
+        'nistk163' : ec.SECT163K1(),
+        'nistp192' : ec.SECP192R1(), 
+        'nistp224' : ec.SECP224R1(),
+        'nistk233' : ec.SECT233K1(),
+        'nistb233' : ec.SECT233R1(),
+        'nistk283' : ec.SECT283K1(),
+        'nistk409' : ec.SECT409K1(),
+        'nistb409' : ec.SECT409R1(),
+        'nistt571' : ec.SECT571K1()
+        }
+
+# The ASN.1 encoded key files use OID instead of common names.
+# Same order as curveTable
+oidTable = {
+    '1.2.840.10045.3.1.7' : ec.SECP256R1(),
+    '1.3.132.0.34' : ec.SECP384R1(),
+    '1.3.132.0.35' : ec.SECP521R1(),
+    '1.3.132.0.1'  : ec.SECT163K1(),
+    '1.2.840.10045.3.1.1' : ec.SECP192R1(),
+    '1.3.132.0.33' : ec.SECP224R1(),
+    '1.3.132.0.26' : ec.SECT233K1(),
+    '1.3.132.0.27' : ec.SECT233R1(),
+    '1.3.132.0.16' : ec.SECT283K1(),
+    '1.3.132.0.36' : ec.SECT409K1(),
+    '1.3.132.0.37' : ec.SECT409R1(),
+    '1.3.132.0.38' : ec.SECT571K1()
+    }
+
+#Return the curve table dictionary for nist name lookup.
+def getCurveTable():
```

Comment:
My programming teacher told me to NEVER directly access variables outside of a class.  He was very strict about this, and for good reason.

But this is python, and normal rules no longer apply. :)